common-app-lib/src/cmr/common_app/services/ingest/subscription_common.clj

```diff
@@ -18,3 +18,14 @@
         (as-> $ (string/join "&" $))
         string/trim
         digest/md5)))
+
+(defn create-query-params
```

Comment:
why was this moved into common?  Is another micro service other than ingest going to use it?  right now I only see ingest using it.   If so then great, if not it probably should remain in ingest.

Reply:
I'm open to discussion on this - my reasoning was that while it _is_ only used in Ingest, it was previously only used in ingest/services while I now needed it in ingest/api. Moving it to common_app/service/ingest seemed like the smart maneuver to cut down on unnecessary imports, since there's no microservice-level common files, but I concede I am not sure if this violates precedent. If it does, I have no issues reverting it.

Reply:
Agree with Erich. Functions used only in ingest should stay in ingest code base. `ingest/api` is supposed to call functions within `ingest/services`, so it is logical to put this function in `ingest/services`.